teachers_digital_platform/__tests__/tdp-analytics.test.js

```diff
@@ -0,0 +1,278 @@
+                    <input type="checkbox" class="a-checkbox" aria-label="Making money" id="topic--making-money" name="topic" value="2">
+                    <label class="a-label" for="topic--making-money">Making money</label>
+                  </div>
+                </li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+
+
+
+      <div id="tdp-search-results">
+        <div class="results_header">
+          <div class="results_count">
+            <h3>Showing 8 matches out of 26 activities</h3>
+          </div>
+          <div class="results_filters">
+            <span class="results_filters-label">Filters applied</span>
+            <div class="results_filters-tags">
+              <div class="a-tag" data-value="#building-block--executive-function" data-js-hook="behavior_clear-filter">
+                Executive function
+                <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 718.9 1200" class="cf-icon-svg"><path d="M451.4 613.7l248.1-248.1c25.6-25.1 26-66.3.8-91.9s-66.3-26-91.9-.8l-.8.8-248.1 248.1-248.1-248.1c-25.4-25.4-66.5-25.4-91.9 0s-25.4 66.5 0 91.9l248.1 248.1L19.5 861.8c-25.6 25.1-26 66.3-.8 91.9s66.3 26 91.9.8l.8-.8 248.1-248.1 248.1 248.1c25.4 25.4 66.5 25.4 91.9 0s25.4-66.5 0-91.9L451.4 613.7z"></path></svg>
+              </div>
+              <div class="a-tag" data-value="#topic--getting-paid" data-js-hook="behavior_clear-filter">
+                Getting paid
+                <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 718.9 1200" class="cf-icon-svg"><path d="M451.4 613.7l248.1-248.1c25.6-25.1 26-66.3.8-91.9s-66.3-26-91.9-.8l-.8.8-248.1 248.1-248.1-248.1c-25.4-25.4-66.5-25.4-91.9 0s-25.4 66.5 0 91.9l248.1 248.1L19.5 861.8c-25.6 25.1-26 66.3-.8 91.9s66.3 26 91.9.8l.8-.8 248.1-248.1 248.1 248.1c25.4 25.4 66.5 25.4 91.9 0s25.4-66.5 0-91.9L451.4 613.7z"></path></svg>
+              </div>
+              <button class="a-btn a-btn__link a-btn__warning a-micro-copy results_filters-clear u-mb10" data-js-hook="behavior_clear-all">Clear all filters</button>
+            </div>
+          </div>
+        </div>
+        <div class="results_list"></div>
+      </div>
+
+    </form>
+  </div>
+`;
+
+const xhr = global.XMLHttpRequest;
+global.console = { error: jest.fn(), log: jest.fn() };
+
+describe( 'The TDP custom analytics', () => {
+
+  beforeEach( () => {
+    // Reset global XHR
+    global.XMLHttpRequest = xhr;
+    // Load HTML fixture
+    document.body.innerHTML = HTML_SNIPPET;
+    // Fire `load` event
+    const event = document.createEvent( 'Event' );
+    event.initEvent( 'load', true, true );
+    window.dispatchEvent( event );
+
+    const mockXHR = {
+      open: jest.fn(),
+      send: jest.fn(),
+      readyState: 4,
+      status: 200,
+      onreadystatechange: jest.fn(),
+      responseText: []
+    };
+    global.XMLHttpRequest = jest.fn( () => mockXHR );
+    spy = jest.spyOn( tdpAnalytics, 'sendEvent' );
```

Comment:
@Chris I've created a spy for tdpAnalytics.sendEvent() that should get triggered  after line 185. But it doesn't seem to be triggering.